### ISSUE TITLE
[SPIKE] to understand what data can be retrieved from RS for destinations

### ISSUE THREAD

**Issue by charlye-skylight** [CDCgov/prime-simplereport#3873]
Upon a submission to RS, we'd like to understand what data we can retrieve from RS.

- submissions API - can we link a destination to a specific test event?
- does a history API exist per test event?
- any additional areas where we might retrieve/obtain data?
- what kinds of errors does the RS API return?

Who from RS should we talk to? 
- engineering channel 
- Jim Duff (secondary option) or Joel B.

Time box to ~8 hours active time.

**Comment by nathancrtr** [comment]
Potentially useful: a little [History API documentation](https://github.com/CDCgov/prime-reportstream/blob/8c4bb2489f04a6f729709c7c9f9411c443e3c626/prime-router/docs/proposals/0015-history-api.md) in plain English

**Comment by DanielSass** [comment]
Addressing direct questions from the spike:
 - submissions API - can we link a destination to a specific test event?

That does not seem possible given the existing report stream api options available.  The report history endpoint has all of the destinations that items from the report were sent to, but doesn't not connect specific test events to their destinations.
 - does a history API exist per test event?

[This report stream proposal](https://github.com/CDCgov/prime-reportstream/blob/8c4bb2489f04a6f729709c7c9f9411c443e3c626/prime-router/docs/proposals/0015-history-api.md#item-resource) makes reference to an endpoint (`GET api/waters/item/{trackingId}/history`) that would support this, but [our conversation](https://skylight-hq.slack.com/archives/C01HFHPNJEL/p1657200997168219?thread_ts=1657125214.903909&cid=C01HFHPNJEL) with report stream engineers indicate that this will not be available anytime soon
 - any additional areas where we might retrieve/obtain data?

specifically for items/test events that were filtered out, we have data about where the item **_didn't_** go.  Which is not what we want necessarily but it seems to bear mentioning
- what kinds of errors does the RS API return?

scheduling a follow up meeting with report stream to cover this as well as some other questions we came across as we dug into this.
